CONTRIBUTING.md

```diff
@@ -1,115 +1,139 @@
+pytest
 
-// Run all the unit tests defined in the specified test file.
-$ pytest tests/${TARGET_TEST_FILE_NAME}
+# Run all the unit tests defined in the specified test file.
+pytest tests/${TARGET_TEST_FILE_NAME}
 ```
 
+## Continuous Integration and Local Verification
 
-### CircleCI Local CLI
+CircleCI is used for continuous integration.
 
-The following circleci job runs all unit tests in Python 3.7:
+### Local Verification
 
-```console
-// Note that this job will download several hundred megabytes of data to
-// install all the packages required for testing,
-// and take several tens of minutes to complete all tests.
-$ circleci build --job tests-python37
+By installing the [`circleci`](https://circleci.com/docs/2.0/local-cli/) local CLI and Docker, you can run tests locally.
+
+```bash
+circleci build --job <job_name>
 ```
 
-You can run tests and examples for each Python version using the following jobs:
+You can run the following jobs.
 
 - `tests-python35`
+  - Runs unit tests under Python 3.5
 - `tests-python36`
+  - Runs unit tests under Python 3.6
 - `tests-python37`
-- `examples-python35`
-- `examples-python36`
-- `examples-python37`
-
-
-In addition, to check the documents, run:
-
-```
-$ circleci build --job document
+  - Runs unit tests under Python 3.7
+- `tests-python38`
+  - Runs unit tests under Python 3.8
+- `checks`
+  - Checks guidelines
+- `document`
+  - Checks documentation build
+- `doctest`
+  - Checks doctest validity
+- `codecov`
+  - Checks unit test code coverage
+
+#### Example
+
+The following `circleci` job runs all unit tests in Python 3.7:
+Note that this job will download several hundred megabytes of data to install all the packages required for testing, and take several tens of minutes to complete all tests.
+
+```bash
+circleci build --job tests-python37
 ```
 
+## Creating a Pull Request
 
-## Documentation
+When you are ready to create a pull request, please try to keep the following in mind.
 
-When adding a new feature to the framework, you also need to document it in the reference.
-The documentation source is stored under [docs directory](./docs) and written in
-[reStructuredText format](http://www.sphinx-doc.org/en/master/usage/restructuredtext/index.html).
+### Title
 
-To build the documentation, you need to install [Sphinx](http://www.sphinx-doc.org):
+The title of your pull request should
 
-```
-$ pip install sphinx sphinx_rtd_theme
-```
+- briefly describe and reflect the changes
+- end with a punctuation
+- wrap any code with backticks
 
-Note that docstrings (documentation comments in the source code) are collected from the installed
-Optuna module. If you modified docstrings, make sure to install the module
-before building the documentation.
+*The title will be directly visible in the release notes.*
 
-```
-$ pip install -e .
-```
+#### Example
 
-Then you can build the documentation in HTML format locally:
+Introduces Tree-structured Parzen Estimator to `optuna.samplers`.
```

Comment:
How about enclosing this with three backticks?

````suggestion
```
Introduces Tree-structured Parzen Estimator to `optuna.samplers`.
```
````